server/base/src/main/java/org/apache/accumulo/server/master/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -254,7 +255,7 @@ public int getMaxConcurrentMigrations() {
   }
 
   @Override
-  public void init(ServerConfiguration conf) {
+  public void init(ServerConfigurationFactory conf) {
```

Comment:
Ah, sorry. Refreshing myself.

IMO, we need to fix the type hierarchy mess that is TableLoadBalancer. I won't push it on to you to fix, but can you please create a JIRA issue to track removing this duplicity and to come up with a plan for compatibility (an interface that lives in public API)? It'd be awesome if you'd want to do it as well :)

I'm ok with this getting merged in.